### DISCUSSION TITLE
Outdoor Cameras

### DISCUSSION THREAD

**Discussion by ragenhe** [mrlt8/docker-wyze-bridge#225] [Q&A]
I've not been able to get my outdoor cam working completely.

It stays up for about two minutes, then the stream dies.

I've tried with the base station on ethernet and wifi, same result.

Anyone made more progress?

**Comment by amzachrich14** [comment]
> I've not been able to get my outdoor cam working completely.
> 
> It stays up for about two minutes, then the stream dies.
> 
> I've tried with the base station on ethernet and wifi, same result.
> 
> Anyone made more progress?

I have an outdoor cam running just fine. Can you send your logs?

**Reply by ragenhe** [reply]
Will do shortly.

**Reply by NotaProgrammer26** [reply]
I've been able to see the Outdoor cameras and stream them.  However I've disabled them, since the streaming drains the battery.  I've got a solar panel hooked up and it kept up for 8 hours.  (Didn't realize it was running, since I normally don't have it sending data to HA.  )   But it will take at least a day to recover and I may have to plug the camera into an outlet to recover it.   So, it works on the current version.

**Reply by mrlt8** [reply]
Interesting. I thought the solar panels might have been enough to keep the streams alive, but I guess that's why wyze hasn't released official RTSP support yet.

Hoping to have on-demand streaming in the next version.

**Reply by ddenson** [reply]
> I've been able to see the Outdoor cameras and stream them. However I've disabled them, since the streaming drains the battery. I've got a solar panel hooked up and it kept up for 8 hours. (Didn't realize it was running, since I normally don't have it sending data to HA. ) But it will take at least a day to recover and I may have to plug the camera into an outlet to recover it. So, it works on the current version.

How did you disable your out door camera? I'm having the same battery drain issues. Thanks in advance!
